src/main/rx/rx.h

```diff
@@ -200,6 +200,9 @@ uint8_t getRssiDbm(void);
 void setRssiDbm(uint8_t newRssiDbm, rssiSource_e source);
 void setRssiDbmDirect(uint8_t newRssiDbm, rssiSource_e source);
 
+void setRFmodeDirect(uint8_t rf_mode_value);
+uint8_t rxGetRFmode(void);
```

Comment:
Function naming should be camelCase - so:
```
setRfModeDirect
```
for example. And the naming would probably make more sense as `rxSetRfMode` and `rxGetRfMode`. In this case there is no "direct" vs. non-direct.

Reply:
fair enough, I wasn't sure of the XXdirect naming convention.